manifests/03-rbac-role-cluster-extensions.yaml

```diff
@@ -9,10 +9,20 @@ rules:
   - console.openshift.io
   resources:
   - consolelinks
-  - consoleclidownloads
   - consolenotifications
   - consoleexternalloglinks
   verbs:
   - get
   - list
   - watch
+- apiGroups:
+  - console.openshift.io
+  resources:
+  - consoleclidownloads
+  verbs:
+  - get
+  - list
+  - watch
+  - create
```

Comment:
This should only be get/list/watch

Reply:
I think he needs full `CRUD` for this one, since the `operator` is creating 2 of this resource, or updating if tampered with.

Reply:
I updated the test incorrectly, added this resource to the same list as the rest of our resources.  My mistake, I forgot this resource is more actively managed than our others.

Reply:
So you probably need to pull the `consoleclidownloads` out of `ClusterRole` `console-extensions-reader` file and put it in its own file. Name the role something like `console-clidownloads-extension-manager` and then create a `ClusterRoleBinding` for `console-operator`.

Then we can update the `origin` test to allow this.

Reply:
No, this is wrong. This is the role that gets assigned to `system:authenticated`. We *cannot* let all users edit these. The operator role is a different one.

Reply:
> I updated the test incorrectly, added this resource to the same list as the rest of our resources. My mistake, I forgot this resource is more actively managed than our others.

Your origin test updates are correct.

Reply:
Right, intended to mean "console operator only" definitely not, "everyone".